tests/test_docstring_reference_parser.py

```diff
@@ -4,125 +4,94 @@
 from r2t2.docstring_reference_parser import (
     DOCSTRING_SHORT_PURPOSE,
     NOTEBOOK_SHORT_PURPOSE,
-    parse_and_add_docstring_references_from_files
+    parse_and_add_docstring_references_from_files,
 )
 
-DOI_URL_HTTPS_PREFIX = 'https://doi.org/'
+DOI_URL_HTTPS_PREFIX = "https://doi.org/"
 HERE = Path(__file__).parent
 FIXTURES = HERE / "fixtures"
 
-DOI_1 = '10.1234/zenodo.1234567'
-DOI_2 = '10.5281/zenodo.1185316'
-DOI_3 = '10.3435/zenodo.1579823'
+DOI_1 = "10.1234/zenodo.1234567"
+DOI_2 = "10.5281/zenodo.1185316"
+DOI_3 = "10.3435/zenodo.1579823"
 
 
 class TestParseAndAddDocstringReferencesFromFiles:
-    def test_should_parse_docstring_reference(
-        self, temp_dir: Path
-    ):
-        file_path = temp_dir / 'test.py'
-        file_path.write_text('\n'.join([
-            'def some_function():'
-            '    """',
-            '    ' + DOI_1,
-            '    """'
-        ]))
-        biblio = Biblio()
-        parse_and_add_docstring_references_from_files(
-            [file_path],
-            biblio=biblio
+    def test_should_parse_docstring_reference(self, temp_dir: Path):
+        file_path = temp_dir / "test.py"
+        file_path.write_text(
+            "\n".join(["def some_function():" '    """', "    " + DOI_1, '    """'])
```

Comment:
just a minor "rant" at using a code formatter. Like here it is loosing the logical formatting that is multiple lines, of course it could have used a multi-line string. It's good to be consistent though. (I just personally don't think it necessarily improves readability)